sdk/cosmos/tests.yml

```diff
@@ -1,339 +1,340 @@
-      TimeoutInMinutes: 240
-      EnvVars:
-        ACCOUNT_HOST: $(java-cosmos-strong-singleregion-host)
-        ACCOUNT_KEY: $(java-cosmos-strong-singleregion-key)
-        SECONDARY_ACCOUNT_KEY: $(java-cosmos-strong-singleregion-secondary-key)
-      MaxParallel: 1
-      Matrix:
-        # 59m 58s, timed out
-        Https_E2E:
-          DisplayName: Single Region ReadMyWrites
-          OSVmImage: 'windows-2019'
-          ProfileFlag: '-Pe2e'
-          AdditionalArgs: '-DargLine="-Dazure.cosmos.directModeProtocol=Https"'
-          DESIRED_CONSISTENCY: 'Strong'
-        # 08m 50s, expect passed
-        Strong_Https_Fast:
-          DisplayName: Strong HTTP Fast
-          OSVmImage: 'macOS-10.13'
-          ProfileFlag: '-Pfast'
-          AdditionalArgs: '-DargLine="-DACCOUNT_DEFAULT_CONSISTENCY=Strong"'
-          PROTOCOLS: '["Https"]'
-          DESIRED_CONSISTENCIES: '["Strong", "Session"]'
-        # 08m 50s, expect passed
-        Strong_Https_Direct:
-          DisplayName: Strong HTTP Direct
-          OSVmImage: 'macOS-10.13'
-          ProfileFlag: '-Pdirect'
-          AdditionalArgs: '-DargLine="-DACCOUNT_DEFAULT_CONSISTENCY=Strong"'
-          PROTOCOLS: '["Https"]'
-          DESIRED_CONSISTENCIES: '["Strong", "Session"]'
-        # 07m 36s, expect passed
-        Strong_Https_Examples:
-          DisplayName: Strong HTTP Examples
-          OSVmImage: 'macOS-10.13'
-          ProfileFlag: '-Pexamples'
-          AdditionalArgs: '-DargLine="-DACCOUNT_DEFAULT_CONSISTENCY=Strong"'
-          PROTOCOLS: '["Https"]'
-          DESIRED_CONSISTENCIES: '["Strong", "Session"]'
-        # 06m 59s, expect passed
-        Strong_Https_NonEmulator:
-          DisplayName: Strong HTTP NonEmulator
-          OSVmImage: 'macOS-10.13'
-          ProfileFlag: '-Pnon-emulator'
-          AdditionalArgs: '-DargLine="-DACCOUNT_DEFAULT_CONSISTENCY=Strong"'
-          PROTOCOLS: '["Https"]'
-          DESIRED_CONSISTENCIES: '["Strong", "Session"]'
+        TestName: Single_Region_Strong_Https
+        ServiceDirectory: cosmos
+        TimeoutInMinutes: 240
+        EnvVars:
+            ACCOUNT_HOST: $(java-cosmos-strong-singleregion-host)
+            ACCOUNT_KEY: $(java-cosmos-strong-singleregion-key)
+            SECONDARY_ACCOUNT_KEY: $(java-cosmos-strong-singleregion-secondary-key)
+        MaxParallel: 1
+        Matrix:
+            # 59m 58s, timed out
+            Https_E2E:
+                DisplayName: Single Region ReadMyWrites
+                OSVmImage: 'windows-2019'
+                ProfileFlag: '-Pe2e'
+                AdditionalArgs: '-DargLine="-Dazure.cosmos.directModeProtocol=Https"'
+                DESIRED_CONSISTENCY: 'Strong'
+            # 08m 50s, expect passed
+            Strong_Https_Fast:
+                DisplayName: Strong HTTP Fast
+                OSVmImage: 'macOS-10.13'
+                ProfileFlag: '-Pfast'
+                AdditionalArgs: '-DargLine="-DACCOUNT_DEFAULT_CONSISTENCY=Strong"'
```

Comment:
I saw you renamed `ACCOUNT_DEFAULT_CONSISTENCY` to `ACCOUNT_CONSISTENCY` above, which was the right thing to do. seems this one is missed.

As per the code in `/sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/TestConfigurations.java` 
The config name is `ACCOUNT_CONSISTENCY` not `ACCOUNT_DEFAULT_CONSISTENCY`


```
    public static String CONSISTENCY =
            System.getProperty("ACCOUNT_CONSISTENCY",
                               StringUtils.defaultString(Strings.emptyToNull(
                                       System.getenv().get("ACCOUNT_CONSISTENCY")), "Strong"));
```